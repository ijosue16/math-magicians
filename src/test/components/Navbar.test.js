import React from 'react';
import '@testing-library/jest-dom';
import { cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import NavBar from './Navbar';

afterEach(cleanup);

describe('render home page', () => {
  test('to match snapshot', () => {
    const tree = renderer.create(<NavBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
