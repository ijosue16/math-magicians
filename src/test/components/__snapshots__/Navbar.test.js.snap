// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home page to match snapshot 1`] = `
<nav
  className=" d-flex justify-content-end align-content-center gap-2 container-fluid py-3 pe-5"
>
  <div
    className="container-fluid "
  >
    <h1>
      Math Magician!
    </h1>
  </div>
  <a
    className="text-decoration-none border-end border-2 border-dark px-2  my-auto"
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
  <a
    className="text-decoration-none border-end border-2 border-dark px-2  my-auto "
    href="/calculator"
    onClick={[Function]}
  >
    Calculator
  </a>
  <a
    className="text-decoration-none   my-auto "
    href="/qoute"
    onClick={[Function]}
  >
    Quote
  </a>
</nav>
`;
